src/vivarium_gates_nutrition_optimization/constants/data_keys.py

```diff
@@ -15,7 +15,7 @@ class __Population(NamedTuple):
     AGE_BINS: str = "population.age_bins"
     DEMOGRAPHY: str = "population.demographic_dimensions"
     TMRLE: str = "population.theoretical_minimum_risk_life_expectancy"
-    BACKGROUND_MORBIDITY: str = "cause.other_causes.disability_weight"
+#    BACKGROUND_MORBIDITY: str = "cause.other_causes.disability_weight"
```

Comment:
Has background morbidity been removed from the model?

Reply:
Effectively yes - we will not be including it in production runs. Though I would like to confirm with @alibow before we delete things as I think we might continue to work with background morbidity in the future.

Reply:
Yeah, agreed!